### PR TITLE
Add suffix to duplicate passthrough fields 

### PR DESCRIPTION
**Pull request by fivetran-jamie**
## PR Overview
**This PR will address the following Issue/Feature:**
Though conversions data lives in the new [source](https://github.com/fivetran/dbt_reddit_ads_source/releases/tag/v0.3.0) report tables we included, there are some legacy fields in the pre-existing reports that users may have included and aliased as `conversions`. This will ensure we avoid duplicate column errors by applying a `_c` suffix to any conversions fields previously included by the user via pass through column variables.

**This PR will result in the following new package version:**

Could be v0.3.1, as this is a fix to the already breaking v0.3.0 release. Though perhaps there's no harm in making it v0.4.0? 

**Please provide the finalized CHANGELOG entry which details the relevant changes included in this PR:**

### Basic Validation
Please acknowledge that you have successfully performed the following commands locally:
- [x] dbt run –full-refresh && dbt test
- [ ] dbt run (if incremental models are present) && dbt test

Before marking this PR as "ready for review" the following have been applied:
- [x] The appropriate issue has been linked, tagged, and properly assigned
- [x] All necessary documentation and version upgrades have been applied
- [x] docs were regenerated (unless this PR does not include any code or yml updates)
- [x] BuildKite integration tests are passing
- [x] Detailed validation steps have been provided below

### Detailed Validation
Please share any and all of your validation steps:

Validation tests passing
image

### If you had to summarize this PR in an emoji, which would it be?
 
🛃

### CODE REVIEW

**Review by fivetran-joemarkiewicz — APPROVED** [review]
LGTM

File: integration_tests/dbt_project.yml [hunk 1]
```diff
@@ -20,6 +20,25 @@ vars:
   
   reddit_ads__ad_conversions_passthrough_metrics: 
     - name: 'view_through_conversion_attribution_window_week'
+  reddit_ads__account_passthrough_metrics:
```
Comment (fivetran-avinash) [inline integration_tests/dbt_project.yml:23]: Did we want to keep these new variable configurations for the default `dbt_project.yml` Buildkite runs, or should we comment them out?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,13 @@
+# dbt_reddit_ads v0.3.1
+[PR #15](https://github.com/fivetran/dbt_reddit_ads/pull/15) includes the following updates:
+
+## Bug Fix
+- Ensures the addition of conversion metrics in [v0.3.0](https://github.com/fivetran/dbt_reddit_ads/blob/main/CHANGELOG.md#dbt_reddit_ads-v030) is truly backwards compatible and will avoid duplicate column errors regardless of any pre-existing configurations.
+  - Specifically, if you were previously utilizing [passthrough column](https://github.com/fivetran/dbt_reddit_ads?tab=readme-ov-file#passing-through-additional-metrics) variables to include fields called `conversions`, `view_through_conversions`, `total_value`, or `total_items`, the package's version of these fields will take precedence. Your fields will be included as well, but suffixed with a `_c`.
+
+## Under the Hood
```
Comment (fivetran-avinash) [inline CHANGELOG.md:8]: We should call out the modifications to the consistency tests as well.

File: README.md [hunk 3]
```diff
@@ -126,6 +126,14 @@ vars:
       - name: "new_custom_field"
         alias: "custom_field"
       - name: "a_second_field"
+    reddit_ads__account_conversions_passthrough_metrics:
```
Comment (fivetran-avinash) [inline README.md:129]: We should probably provide an example of how the alias would be applied.

File: README.md
Comment (fivetran-avinash) [inline README.md:1]: @fivetran-jamie I'm wondering if we should call out the backwards-compatibility functionality of conversions fields,  or whether it should live in the DECISIONLOG. This update might not be quite intuitive to a customer unless they really go hunting for the macro.

**Review by fivetran-avinash — COMMENTED** [review]
@fivetran-jamie left a few comments!

File: integration_tests/dbt_project.yml (hunk 1 above)
Comment (fivetran-jamie) [reply integration_tests/dbt_project.yml:23]: Yeah just wanted to make sure this passes on all warehouses. 

What do you mean by source file?

File: integration_tests/dbt_project.yml [hunk 4]
```diff
@@ -20,6 +20,25 @@ vars:
...
+    - name: app_install_metrics_install
+      alias: installs
+    - name: total_items
+  reddit_ads__ad_passthrough_metrics:
+    - name: conversions
+    - name: app_install_metrics_install
+      alias: installs
 
   # For validation testing
   reddit_ads__conversion_event_types:
```
[the remark this answers is not in the document]
Comment (fivetran-jamie) [reply integration_tests/dbt_project.yml:44]: no harm in leaving them IMO, more testing

File: README.md (hunk 3 above)
Comment (fivetran-jamie) [reply README.md:129]: added!

File: README.md
Comment (fivetran-jamie) [reply README.md:1]: That's a good callout -- I think the decision log makes the most sense

File: integration_tests/dbt_project.yml (hunk 1 above)
Comment (fivetran-avinash) [reply integration_tests/dbt_project.yml:23]: Bleh, `dbt_reddit_ads_source`, just so the `dbt_project.yml` files are matching.

Something to do later though!

Comment (fivetran-jamie) [reply integration_tests/dbt_project.yml:23]: Ah gotcha, yah later haha

**Review by fivetran-avinash — APPROVED** [review]
@fivetran-jamie Approved! 

Two final suggestions: Can you add a Documentation Update section to the CHANGELOG and link to the DECISIONLOG there? (Optional, but you could also link to the README as well).

**Pull request merged**